### ISSUE TITLE
Fullscreen options

### ISSUE THREAD

**Issue by CookPete** [CookPete/react-player#33]
After seeing @scpaye506's fork that [enables fullscreen on Vimeo](https://github.com/scpaye506/react-player/commit/fab14d52fefacd445aa34090c394ca0b16e87627), I wonder if we should enable fullscreen for Vimeo by default? It could still be overridden back to `false` via the `vimeoConfig` prop. If not, I would be inclined to disable fullscreen on YouTube, to align behaviour across players.

Even fullscreen for `FilePlayer` is possible, but would require `controls` to be enabled.

Perhaps there is a need for a top level `allowFullScreen` prop?

**Comment by scpaye506** [comment]
I think enabling it by default would be the way to go and then having an option to override. My issue was with how android doesn't open the video into full screen unless allowFullScreen is there, which I feel is expected behavior for users.

**Comment by CookPete** [comment]
This is fixed in `v0.5.2`. Fullscreen is now enabled by default for Vimeo, and can be turned off via config prop:
```
<ReactPlayer
  url='http://vimeo.com/...'
  vimeoConfig={{ iframeParams: { fullscreen: 0 } }}
/>
```

**Comment by CookPete** [comment]
It looks like this is no longer an option: https://github.com/vimeo/player.js#embed-options
